idseq_dag/util/s3.py

```diff
@@ -397,9 +397,9 @@ def fetch_from_s3(src,  # pylint: disable=dangerous-default-value
                     if os.path.exists(tmp_dst):
                         command.remove_rf(tmp_dst)
                     if okay_if_missing:
-                        script = r'aws s3 cp --quiet "${src}" - ' + command_params
+                        script = r'set -o pipefail; aws s3 cp --quiet "${src}" - ' + command_params
```

Comment:
does this preserve the intention of `okay_if_missing`? just want to be sure

Reply:
Thanks for flagging! You must be right about the `okay_if_missing`. Will fix

Reply:
Added a comment and removed it from the `okay_if_missing` case

Reply:
Now that I read this more carefully, it looks like it /is/ correct to have `set -o pipefail` on both. `okay_if_missing` is checked again in the exception handler, so that's consistent.